docs/deployment/components.md

```diff
@@ -0,0 +1,58 @@
+---
+myst:
+  html_meta:
+    "description": "Components of a Plone 6 application for deployment"
+    "property=og:description": "Components of a Plone 6 application for deployment"
+    "property=og:title": "Components of a Plone 6 application for deployment"
+    "keywords": "Plone, deployment, components, backend, volto, fronted, TLS termination proxy, load balancer, router, database, mail service, optimization, maintenance"
+---
+
+(deployment-components-label)=
+
+# Components
+
+This page in the deployment guide covers the components in a Plone application deployment.
+The components can be broken down into those that are required, recommended, and optional.
+
+
+(deployment-required-components-label)=
+
+## Required components
+
+You need the following components to deploy a Plone application.
+
+-   {term}`Plone backend`
+-   {term}`Volto` as the default {term}`Frontend` for Plone
+-   {term}`TLS termination proxy`
+-   {term}`Load balancer` or router
+-   Database
+-   Mail service
+-   Optimization tasks
```

Comment:
This is not a component. We should have a separate chapter "Optimize and tune your Plone site" which explains how to monitor and tune.

Reply:
Done in e3f4a1666941487da502b02607eb0a33d54ef103